master/buildbot/newsfragments/gitlab_ref.bugfix

```diff
@@ -0,0 +1 @@
+Fixed `ref` to point to `branch` instead of commit `sha` in `:py:class:`~buildbot.reporters.GitlabStatusPush`
```

Comment:
it is without the first `

    :py:class:`~buildbot.reporters.GitlabStatusPush`